modules/ROOT/pages/dataweave-formats.adoc

```diff
@@ -738,54 +681,73 @@ See xref:6@studio::input-output-structure-transformation-studio-task.adoc[To Def
 
-For example, assume that you have a copybook definition file (`mailing-record.cpy`) that looks like this:
+For example, assume that you have a copybook definition file
+(`mailing-record.cpy`) that looks like this:
 
-.Example: Copybook Definition
 ----
-01  MAILING-RECORD.
-    05  COMPANY-NAME            PIC X(30).
-    05  CONTACTS.
-        10  PRESIDENT.
-            15  LAST-NAME       PIC X(15).
-            15  FIRST-NAME      PIC X(8).
-        10  VP-MARKETING.
-            15  LAST-NAME       PIC X(15).
-            15  FIRST-NAME      PIC X(8).
-        10  ALTERNATE-CONTACT.
-            15  TITLE           PIC X(10).
-            15  LAST-NAME       PIC X(15).
-            15  FIRST-NAME      PIC X(8).
-    05  ADDRESS                 PIC X(15).
-    05  CITY                    PIC X(15).
-    05  STATE                   PIC XX.
-    05  ZIP                     PIC 9(5).
+       01  MAILING-RECORD.
+           05  COMPANY-NAME            PIC X(30).
+           05  CONTACTS.
+               10  PRESIDENT.
+                   15  LAST-NAME       PIC X(15).
+                   15  FIRST-NAME      PIC X(8).
+               10  VP-MARKETING.
+                   15  LAST-NAME       PIC X(15).
+                   15  FIRST-NAME      PIC X(8).
+               10  ALTERNATE-CONTACT.
+                   15  TITLE           PIC X(10).
+                   15  LAST-NAME       PIC X(15).
+                   15  FIRST-NAME      PIC X(8).
+           05  ADDRESS                 PIC X(15).
+           05  CITY                    PIC X(15).
+           05  STATE                   PIC XX.
+           05  ZIP                     PIC 9(5).
 ----
-Note: Copybook definitions must always begin with a `01` entry. A separate record type is generated for each `01` definition in your copybook (there must be at least one 01 definition for the copybook to be usable, so add one using an arbitrary name at the start of the copybook if none is present). If there are multiple `01` definitions in the copybook file, you can select which definition to use in the transform from the dropdown list.
 
-When you import the schema, the Transform component converts the copybook file to a flat file schema that it stores in the `src/main/resources` folder of your Mule project. In flat file format, the copybook definition above looks like this:
+* Copybook definitions must always begin with a `01` entry. A separate record
+  type is generated for each `01` definition in your copybook (there must be at
+  least one `01` definition for the copybook to be usable, so add one using an
+  arbitrary name at the start of the copybook if none is present). If there are
+  multiple `01` definitions in the copybook file, you can select which
+  definition to use in the transform from the dropdown list.
+* COBOL format requires definitions to only use columns 7-72 of each line. Data
+  in columns 1-5 and past column 72 is ignored by the import process. Column 6
```

Comment:
"The import process ignores data..."

Reply:
We want to emphasize what's ignored here.